src/server/lib/getConfiguration.ts

```diff
@@ -184,6 +184,12 @@ export const getConfiguration = (): Configuration => {
       process.env.OKTA_CLIENT_SECRET,
       'OKTA client secret missing',
     ),
+    groupIds: {
+      GuardianUserAll: getOrThrow(
+        process.env.OKTA_GUARDIAN_USERS_GROUP_ID,
+        'OKTA_GUARDIAN_USERS_GROUP_ID missing',
+      ),
+    },
```

Comment:
I't annoying to have to add this to the configuration. I guess the alternative would be to make an extra api call on startup to get the group id from its name. But I don't know if it's worth the trouble

Reply:
agreed, I don't think it'd be worth the trouble in this case either. in my mind it would be more appealing if we started to need multiple group ids in the future (for example, the Jobs user group id could foreseeably be needed)